controllers/devworkspace/controller.go

```diff
@@ -87,6 +87,13 @@ func GetCurrentCheClusterInstances() map[client.ObjectKey]v2alpha1.CheCluster {
 	return ret
 }
 
+func CleanCheClusterInstancesForTest() {
```

Comment:
Could we move this to test file?

Reply:
This needs to access data private to the devworkspace package but needs to be called from test code in other packages. Moving it to `controller_test.go` makes it invisible to other packages unfortunately.